ansible_mitogen/process.py

```diff
@@ -285,8 +285,10 @@ class Broker(mitogen.master.Broker):
     the exuberant syscall expense of EpollPoller, so override it and restore
     the poll() poller.
     """
-    poller_class = mitogen.core.Poller
```

Comment:
I've just noticed the above docstring mentions "does not need the exuberant syscall expense of EpollPoller, so override it and restore the poll() poller" written by dw in https://github.com/mitogen-hq/mitogen/commit/7b129e857627ca38d1ebc38b4b4396e8d560d7c1. That predates Poller switching from `select.poll()` to `select.select()` in https://github.com/mitogen-hq/mitogen/commit/9bcd2ec56c3162fc5455effa6ffc178c690ca2f1.

Your approach using if/else, instead of `PREFERRED_POLLER` makes more sense now.